statistics.md

```diff
@@ -0,0 +1,93 @@
+# Collecting user interactions
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Views endpoint
+**POST /api/statistics/views**
+
+To track a page view, make an HTTP POST request to this endpoint.
+
+The sections below describe the parameters for a page view request. All other information about the interaction can be derived from the request itself.
+
+### Parameters
+- `targetId`: The id of the object the user viewed
+- `targetType`: The type of the object the user viewed
+
+### Status codes:
+
+- `201` Created: if the operation succeeded
+- `422` Unprocessable Entity: if any of the parameters are missing or invalid.
```

Comment:
400 should be more appropriate here as the parameters are mandatory. 422 should be used if the parameter are provided, syntactically valid, by leading to not existing resources

Reply:
Agreed